Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -83,7 +85,8 @@ static ManagementClient()
             int portNumber = 15672,
             TimeSpan? timeout = null,
             Action<HttpRequestMessage> configureRequest = null,
-            bool ssl = false)
+            bool ssl = false,
+            Action<HttpClientHandler> handlerOverride = null)
```

Comment:
If we talk about **overriding** handler then consider to change 
```c#
Action<HttpClientHandler> handlerOverride = null
```
to
```c#
Func<HttpClientHandler, HttpClientHandler> handlerOverride = handler => handler
```

Reply:
It's not overriding in this case, it's configuring I think? @guojianwei001 could you rename it to handlerConfigurator instead of handlerOverride maybe?

Reply:
Yes, you are right. I changed it to handlerConfigurator.